Packs/VaronisSaaS/Integrations/VaronisSaaS/VaronisSaaS.py

```diff
@@ -0,0 +1,2108 @@
+        if start_time and end_time:
+            time_condition = time_condition\
+                .set_operator("Between")\
+                .add_value({event_attributes.Event_TimeUTC: start_time.isoformat(),
+                            f"{event_attributes.Event_TimeUTC}0": end_time.isoformat()})
+            # "displayValue": start_time.isoformat(), (this line seems to be commented out)
+        if last_days:
+            time_condition\
+                .set_operator("LastDays")\
+                .add_value({event_attributes.Event_TimeUTC: last_days, "displayValue": last_days})
+        search_request.query.filter.add_filter(time_condition)
+
+        if descending_order == 'True':
+            search_request.rows.add_ordering({"path": event_attributes.Event_TimeUTC, "sortOrder": "Desc"})
+        else:
+            search_request.rows.add_ordering({"path": event_attributes.Event_TimeUTC, "sortOrder": "Asc"})
+
+        dataJSON = search_request.to_json()
+
+        create_search = self._http_request(
+            'POST',
+            '/app/dataquery/api/search/v2/search',
+            data=dataJSON,
+            headers=self.headers
+        )
+
+        url = create_search[0]["location"]
+        json_data = self._http_request(
+            method='GET',
+            url_suffix=f'/app/dataquery/api/search/{url}',
+            headers=self.headers,
+            status_list_to_retry=[304, 405, 206],
+            retries=10
+        )
+
+        mapper = SearchEventObjectMapper()
+        events = mapper.map(json_data)
+        return events
+
+    def varonis_get_enum(self, enum_id: int) -> List[Any]:
+        """Gets an enum by enum_id. Usually needs for retrieving object required for a search
+
+        :type enum_id: ``int``
+        :param enum_id: Id of enum stored in database
+
+        :return: The list of objects required for a search filter
+        :rtype: ``List[Any]``
+        """
+        response = self._http_request('GET', f'/api/entitymodel/enum/{enum_id}', headers=self.headers)
+        return response
+
+    def varonis_update_alert_status(self, query: Dict[str, Any]) -> bool:
+        """Update alert status
+
+        :type query: ``Dict[str, Any]``
+        :param query: Update request body
+
+        :return: Result of execution
+        :rtype: ``bool``
+
+        """
+        return self._http_request(
+            'POST',
+            '/api/alert/alert/SetStatusToAlerts',
+            json_data=query,
+            headers=self.headers)
+
+    def varonis_add_note_to_alerts(self, query: Dict[str, Any]) -> bool:
+        """Update alert status
+
+        :type query: ``Dict[str, Any]``
+        :param query: "add notes" request body
+
+        :return: Result of execution
+        :rtype: ``bool``
+
+        """
+        return self._http_request(
+            'POST',
+            '/api/alert/alert/AddNoteToAlerts',
+            json_data=query,
+            headers=self.headers)
+
+
+class EventAttributes:
```

Comment:
Instead of the class, we can just ise a simple method to convert `_` to `.`
Not mandatory, but will make the code much easier to read

Reply:
I see. I've got your point but if you don't mind we will leave as is.
We have to support a dynamic output (return additional columns if requested by user) so we must have a full list of mandatory and extra fields. We just don't want to introduce some major changes in it now.